### ISSUE TITLE
Typing `shift+s` enters "ShiftS" instead of just "S"

### ISSUE THREAD

**Issue by jkroso** [atom/atom#13172]
I tried installing an older version of Atom and it worked fine there so it must be a recent change

### Versions

```
$ atom --version
Atom    : 1.12.0
Electron: 1.3.6
Chrome  : 52.0.2743.82
Node    : 6.3.0
```

```
$ apm --version
apm  1.12.9
npm  3.10.5
node 4.4.5
python 2.7.11
git 2.9.2
```

Running OSX 10.12.1

**Comment by darklight721** [comment]
Might be related to this issue, but mine enters 'Alt' when I do `alt-left` or `alt-right`.

Here are my keybindings for `alt-left`:
```
alt-left	native!	Core	body .native-key-bindings
alt-left	editor:move-to-beginning-of-word	Core	atom-text-editor
alt-left	tree-view:recursive-collapse-directory	Tree View	.tree-view
```

Got the same versions as above.

**Comment by jkroso** [comment]
I should mention it's not just with "s" that it does it. It does it with
all keys but only the first time the letter is pushed. E.g Shift+s+s enters
in "ShiftSS"

**Comment by Ben3eeE** [comment]
Can you enter the following code in to the dev tools console and post the results when you press these keys.

```
document.addEventListener('keydown', e => console.log(e), true)
```

What keyboard layout are you using?

@darklight721 Are you also on mac?

**Comment by darklight721** [comment]
@Ben3eeE Yes I'm also on mac (10.12).

Here are the results:
![screen shot 2016-11-10 at 4 31 35 pm](https://cloud.githubusercontent.com/assets/577701/20169613/6e6dd0ba-a763-11e6-8e7e-dac11a339978.png)
![screen shot 2016-11-10 at 4 32 02 pm](https://cloud.githubusercontent.com/assets/577701/20169616/7a1b2aac-a763-11e6-9798-ffdc1274975c.png)

**Comment by jkroso** [comment]
Heres me pushing shift+s

![image](https://cloud.githubusercontent.com/assets/1103796/20170183/2a012b98-a790-11e6-9d58-4076c7e12b0a.png)

**Comment by Ben3eeE** [comment]
Just to confirm. Are these results from safe mode of Atom `atom --safe`? I know there were similar reports were dead keys would type `dead` in the editor when using the now deprecated `keyboard-localization` package.

**Comment by darklight721** [comment]
@Ben3eeE I just tried it on safe mode and it's still the same.

**Comment by 50Wliu** [comment]
By keyboard layout, @Ben3eeE is referring to the localization of the keyboard (eg Polish keyboard, Swedish, German...).

**Comment by nathansobo** [comment]
@ungb heads up on this one. I can't figure out how to reproduce it. Can any maintainers repro?

**Comment by Ben3eeE** [comment]
Can't repro on Windows 7 64bit or 10 anniversary 64bit. Maybe mac only?

**Comment by jkroso** [comment]
Do your key events look the same as mine?

**Comment by Ben3eeE** [comment]
Yeah looks the same. What is the localization of the keyboard layout that you are using (eg Polish keyboard, Swedish, German, etc...)? Are you using some special input methods?

**Comment by jkroso** [comment]
Australian I think. I use some key remapping software but I turned it off
and no other apps were having this issue anyway

**Comment by nathansobo** [comment]
Sorry about this. This is caused by a bug in keystroke replay that made it through our beta cycle, and we're actively working on a hotfix right now.

**Comment by nathansobo** [comment]
Fixed via https://github.com/atom/atom-keymap/issues/171 and available in 1.12.1. Sorry about this issue everybody.
